Library/Homebrew/gpg.rb

```diff
@@ -1,26 +1,10 @@
-require "utils"
-
 class Gpg
-  def self.find_gpg(executable)
-    which_all(executable).detect do |gpg|
-      gpg_short_version = Utils.popen_read(gpg, "--version")[/\d\.\d/, 0]
-      next unless gpg_short_version
-      gpg_version = Version.create(gpg_short_version.to_s)
-      @version = gpg_version
-      gpg_version >= Version.create("2.0")
-    end
-  end
-
-  def self.executable
-    find_gpg("gpg") || find_gpg("gpg2")
-  end
-
   def self.available?
-    File.executable?(executable.to_s)
+    File.executable?(GPG2Requirement.new.executable.to_s)
```

Comment:
I'd rather not have requirements getting called by utility code this way.

Reply:
Alright, I can rework it to not call the requirement, that's not a major issue.

Reply:
Cool 👍